### DISCUSSION TITLE
PHP8 support

### DISCUSSION THREAD

**Discussion by martynishin** [spatie/laravel-medialibrary#2443] [Ideas]
When I try to update composer with PHP8 and spatie/laravel-medialibrary 8.7, then I get an error:
<img width="953" alt="Снимок экрана 2021-05-18 в 17 23 42" src="https://user-images.githubusercontent.com/20871977/118668722-cd113d00-b7fd-11eb-8eeb-959a86551345.png">

**Comment by MikeMazalahi** [comment]
@martynishin I've come to understand that Spatie added support for PHP8 in media-library v9, is there a reason why you can't just edit your composer.json and upgrade to the new version? (or otherwise, I'd downgrade to PHP 7.4)
